server/src/main/java/org/elasticsearch/cluster/coordination/indicators/LackOfQuorumIndicator.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.cluster.coordination.indicators;
+
+import org.elasticsearch.cluster.node.DiscoveryNode;
+import org.elasticsearch.cluster.service.ClusterService;
+import org.elasticsearch.health.HealthIndicatorResult;
+import org.elasticsearch.health.HealthIndicatorService;
+import org.elasticsearch.health.HealthStatus;
+
+import java.util.stream.Collectors;
+
+public class LackOfQuorumIndicator implements HealthIndicatorService {
+
+    public static final String LACK_OF_QUORUM = "lack_of_quorum";
```

Comment:
`lack_of_quorum: green` <- lack_of_quorum is bad, but green is good. I am confused.
May be `can_form_quorum`?

Reply:
Very good point! I've renamed the class and the indicator.